magento1-module/app/code/local/Divante/VueStorefrontBridge/Helper/Address.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+class Divante_VueStorefrontBridge_Helper_Address extends Mage_Core_Helper_Abstract
+{
+    /**
+     * @param array $streetData
+     *
+     * @return string
+     */
+    public function concatStreetData(array $streetData): string
+    {
+        $concatStreetData = '';
+
+        if (isset($streetData[0], $streetData[1])) {
```

Comment:
Magento allows for 4 address lines, can we also do this here to make it compatible?

Reply:
We can to it but as the the reference in [here](https://github.com/DivanteLtd/magento1-vsbridge/blob/master/doc/VueStorefrontBridge%20API%20specs.md#post-vsbridgeuserme) just shows 2. I have just Implement two. 

But i think that should be done in a later PR. As this is the logic which I have mostly tested.